src/utils/intersection-observer-polyfill.js

```diff
@@ -100,8 +101,15 @@ export function getIntersectionChangeEntry(element, owner, hostViewport) {
  * @return {boolean}
  */
 export function nativeIntersectionObserverSupported(win) {
+  if (!('IntersectionObserver' in win)) {
+    return false;
+  }
+  // eslint-disable-next-line no-undef
+  if (INTERSECTION_OBSERVER_POLYFILL || getMode().localDev || getMode().test) {
```

Comment:
Since we won't be able to run experiment with inabox the same time as AMP docs, what about introducing a different experiment flag for the inabox related changes only. 
For example `INTERSECTION_OBSERVER_POLYFILL_INABOX`, then you can specify the default value of the const to `false` [here](https://github.com/ampproject/amphtml/blob/master/build-system/global-configs/experiments-const.json). 
Once we are about to run an experiment with inabox, you can then declare an experiment slot similar to what you have
https://github.com/ampproject/amphtml/blob/ab03e7c87443b5c1f69d4de1e409202475f5b238/build-system/global-configs/experiments-config.json#L2-L8

Reply:
caveat is that you won't get the e2e test coverage.

Reply:
Created a new constant for inabox. PTAL.